### PR TITLE
MBS-12064: Add jazzmusicarchives.com to the database whitelist

### PR DESCRIPTION
**Pull request by reosarevok**
### Implement MBS-12064

### CODE REVIEW

File: root/static/scripts/edit/URLCleanup.js [hunk 1]
```diff
@@ -2337,6 +2337,34 @@ const CLEANUPS: CleanupEntries = {
...
+              result: type === 'artist',
+              target: ERROR_TARGETS.ENTITY,
+            };
+          case LINK_TYPES.otherdatabases.release_group:
+            return {
+              result: type === 'album',
+              target: ERROR_TARGETS.ENTITY,
+            };
+        }
+        return {result: false, target: ERROR_TARGETS.ENTITY};
```
Comment (yvanzo) [inline root/static/scripts/edit/URLCleanup.js:2363]: The Metal Archives are allowed to link to labels and releases. Should Jazz Music Archives be the same?

File: root/static/scripts/tests/Control/URLCleanup.js [hunk 2]
```diff
@@ -2240,6 +2240,21 @@ const testData = [
     expected_relationship_type: ['downloadfree', 'streamingfree'],
             expected_clean_url: 'http://www.jamendo.com/album/56372',
   },
+  // Prog Archives
```
Comment (yvanzo) [inline root/static/scripts/tests/Control/URLCleanup.js:2243]: Typo from copy/paste:
```suggestion
  // Jazz Music Archives
```

**Review by yvanzo — COMMENTED** [review]
Details about which entity types may be linked (or not linked if something has to be specifically avoided) and example links would be welcome in the ticket’s description, for editors not to dig the code. Not sure about labels and releases. The rest looks good!

File: root/static/scripts/edit/URLCleanup.js (hunk 1 above)
Comment (reosarevok) [reply root/static/scripts/edit/URLCleanup.js:2363]: Jazz Music Archives does not seem to have label pages, so those are out. 

http://www.jazzmusicarchives.com/album/john-coltrane/my-favorite-things (for example) does say "Atlantic 1361(US)" which makes it seem like these map to releases, but http://www.jazzmusicarchives.com/album/miles-davis/kind-of-blue for example says "Sony CD Reissue in 1997 includes 1 bonus track" and nothing else - they don't seem to have an entry for said reissues. So I think RG is probably the most reasonable choice to map to.

Comment (yvanzo) [reply root/static/scripts/edit/URLCleanup.js:2363]: Ok thanks, just asking because it is the same network of websites.

Comment (reosarevok) [reply root/static/scripts/edit/URLCleanup.js:2363]: Actually, I think the original user was a bit confused - the site in the same network is http://www.metalmusicarchives.com/ as they linked, but we do not have that - we have https://www.metal-archives.com/ which is a completely different one :) People are just not very original.

**Review by yvanzo — APPROVED** [review]
:ship: 

It’s part of the STYLE workflow in Jira, but it’s not mentioned in the pull request, so just a friendly reminder: The documentation page about other databases has to be updated and will have to be transcluded at release time.

**Comment by reosarevok** [comment]
Updated the page, now I need to remember to transclude.

**Pull request merged**